src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/wwwroot/index.html

```diff
@@ -0,0 +1,95 @@
+                        <form class="form-inline">
+                            <div class="input-prepend input-append">
+                                <input type="text" name="user-name" id="user-name" placeholder="Type your user name" />
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-12">
+                        <form class="form-inline">
+                            <div class="input-prepend input-append">
+                                <input type="text" name="group-name" id="group-name" placeholder="Type a group name" />
+                                <input type="button" id="join-group" class="btn btn-sm btn-primary" value="Join Group" />
+                                <input type="button" id="leave-group" class="btn btn-sm btn-primary" value="Leave Group" />
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-12">
+                        <form class="form-inline">
+                            <div class="input-prepend input-append">
+                                <input type="text" name="group-message" id="group-message-text" placeholder="Type a message" />
+                                <input type="button" id="groupmsg" class="btn btn-sm btn-primary" value="Send to Group" />
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-12">
+                        <hr />
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-6">&nbsp;</div>
+                    <div class="col-6">
+                        <ul id="messagesList"></ul>
+                    </div>
+                </div>
+            </div>
+        </main>
+    </div>
+
+    <footer class="border-top footer text-muted">
+        <div class="container">
+            &copy; 2019 - SignalRTemplate
+        </div>
+    </footer>
+    
+    <!-- jQuery is only being used here to enable advanced Bootstrap features. 
+         jQuery is not a requirement for ASP.NET Core SignalR. -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"
+            crossorigin="anonymous"
+            integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=">
+    </script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/js/bootstrap.bundle.min.js"
+            crossorigin="anonymous"
+            integrity="sha256-E/V4cWE4qvAeO5MOhjtGtqDzPndRO1LBk8lJ/PR7CA4=">
+    </script>
+    <script src="/js/signalr.js"></script>
+    <script src="/js/site.js"></script>
+    <script src="/js/chat.js"></script>
+    
+</body>
+</html>
```

Comment:
While everyone else is giving you feedback I'll just say we should have a newline at the end of all these files.